Sources/PIALibrary/Common/Data/Networking/NetworkRequestClient.swift

```diff
@@ -106,10 +106,14 @@ private extension NetworkRequestClient {
         }
     }
     
-    func getEndpoints(for module: NetworkRequestModule) -> [PinningEndpoint] {
-        switch module {
-        case .account: 
+    func getEndpoints(for module: NetworkRequestModule, environment: Client.Environment = Client.environment) -> [PinningEndpoint] {
```

Comment:
We use the environment to calculate the available endpoints and being able to use staging